AWSAuthSDK/Tests/AWSMobileClientTests/AWSMobileClientTests.swift

```diff
@@ -757,7 +790,57 @@ class AWSMobileClientTests: XCTestCase {
             downloadExpectation.fulfill()
         }
         wait(for: [downloadExpectation], timeout: 5)
-
+        
     }
+    
+}
 
+extension AWSMobileClientError {
```

Comment:
Will this be useful if it's part of the SDK?

Reply:
I thought about it, but I'd prefer not to clutter the API at this point.